index.js

```diff
@@ -15,7 +15,11 @@ var PLUGIN_NAME = 'gulp-sass';
 var gulpSass = function gulpSass(options, sync) {
   return through.obj(function(file, enc, cb) {
     var opts,
+        ip,
         filePush,
+        sassMap,
+        sassMapFile,
+        sassFileSrc,
```

Comment:
Yes. Variable declarations at the top of function scopes, but I realize I have a function scope on l59 that I can (and should) use. Updating.